FLEDGE.md

```diff
@@ -323,8 +344,10 @@ The function gets called once for each candidate ad in the auction.  The argumen
 *   browserSignals: An object constructed by the browser, containing information that the browser knows and which the seller's auction script might want to verify:
     ```
     { 'topWindowHostname': 'www.example-publisher.com',
+      'requestedSize': {width: 100, height: 200}, /* if specified in the bid */
```

Comment:
does this come from the bid or the auction config?

Reply:
I'm curious how `requestedSize` can be optional in the `auctionConfig`.  How do buyers determine whether candidate ads will fit within the slot?

>The optional `requestedSize` field recommends a frame size for the auction, which will be available to bidders in browser signals. This size should be specified in the same format as the sizes in the `adSizes` field of `joinAdInterestGroup`. For convenience, the returned fenced frame config will automatically populate a `<fencedframe>`'s `width` and `height` attributes with the `requestedSize` when loaded, though the element's size attributes can still be modified if you want to change the element's container size. Bidders inside the auction may pick a different content size for the ad, and that resulting size will be visually scaled to fit inside the element's container size.

Reply:
@JensenPaul Good catch, should be auction config.

Reply:
@dmdabbs They can use whatever method they've been using up until now (presumably some unstructured field in publisher signals).

`requestedSize` is just a convenience feature so that the fenced frame config returned from the auction "knows" what size it should set the element to, instead of the caller having to do it manually after the auction. And if the caller is putting the size into `requestedSize`, we don't want them to have to put it redundantly anywhere else (but they still could, if they wanted to).